pkg/testutils/mockmaps/lbmap.go

```diff
@@ -238,3 +244,19 @@ func (m *LBMockMap) UpdateSourceRanges(revNATID uint16, prevRanges []*cidr.CIDR,
 func (m *LBMockMap) DumpSourceRanges(ipv6 bool) (datapathTypes.SourceRangeSetByServiceID, error) {
 	return m.SourceRanges, nil
 }
+
+func (m *LBMockMap) ExistsSockRevNat(cookie uint64, addr net.IP, port uint16) bool {
+	if addr.To4() != nil {
+		key := lbmap.NewSockRevNat4Key(cookie, addr, port)
+		if _, ok := m.SockRevNat4[*key]; ok {
+			return true
+		}
+	} else {
```

Comment:
this has a risk if the addr `net.IP`has not been validated, because it can be nil

Reply:
This is a test util code, so I'm not sure if this is required. But it doesn't hurt to do the validation I suppose.